eos-update-flatpak-repos

```diff
@@ -1536,9 +1537,12 @@ def _migrate_installed_flatpaks():
 
 
 if __name__ == '__main__':
-    # Send logging messages both to the console and the journal
-    logging.basicConfig(level=logging.INFO)
-    logging.root.addHandler(journal.JournalHandler())
+    # Send logging messages to the journal, and to stderr if it's not the
+    # journal.
+    handlers = [journal.JournalHandler()]
+    if not GLib.log_writer_is_journald(sys.stderr.fileno()):
```

Comment:
```suggestion
    if sys.stderr.isatty():
```

Reply:
You could imagine wanting to run this through (eg) `| grep kbieberling`?

Reply:
oh, stdout ≠ stderr